thread-frontend/.output/public/.vite/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Add `thread-frontend/.output` to the git ignore and remove these files from git/jj